### PR TITLE
build: add qvge

### PR DESCRIPTION
**Pull request by Van020530**
截图_选择区域_20231023231422

QVGE is a multiplatform graph editor written in C++/Qt. Its main goal is to make possible visually edit two-dimensional graphs in a simple and intuitive way.

### CODE REVIEW

File: io.github.qvge/linglong.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  kind: qmake 
+  manual:
+    configure: |
+      cd src
+      qmake -makefile  ${conf_args} ${extra_args}
+    build: |
+      make ${jobs}
+    install: |
+      make ${jobs} DESTDIR=${dest_dir} install
+      cp  /source/src/bin/qvge ${PREFIX}/bin/qvgeapp
```
Comment (kamiyadm) [inline io.github.qvge/linglong.yaml:29]: 用patch来修改一下

Comment (Van020530) [reply io.github.qvge/linglong.yaml:29]: image
似乎他不是没有文件，我刚刚把cp去掉之后再跑一边也能run出来。

Comment (Van020530) [reply io.github.qvge/linglong.yaml]: 是不是我在longlong.yml中把上面kind设置为lib的时候就没有opt/apps那个目录

Comment (kamiyadm) [reply io.github.qvge/linglong.yaml]: lib的时候是/runtime, app的时候是 /opt/apps

**Review by kamiyadm — APPROVED** [review]

**Pull request merged**